VERSION.txt

```diff
@@ -1 +1 @@
-0.0.6
+0.1.0
```

Comment:
This is a breaking change to the interface, so incrementing the minor version number seems appropriate.